pulsar-common/src/test/java/org/apache/pulsar/common/naming/TopicNameTest.java

```diff
@@ -162,6 +161,9 @@ public void topic() {
         assertEquals(TopicName.get("persistent://tenant/cluster/namespace/topic")
                 .getPersistenceNamingEncoding(), "tenant/cluster/namespace/persistent/topic");
 
+        assertEquals(TopicName.fromPersistenceNamingEncoding("tenant/cluster/namespace/persistent/topic"),
```

Comment:
You may need to add a case for using the `getPersistenceNamingEncoding` returned value to parse to ensure the encoded value can be parsed successfully.